aten/src/ATen/native/vulkan/VulkanAten.cpp

```diff
@@ -272,6 +272,18 @@ Tensor vulkan_add(const Tensor& self, const Tensor& other, const Scalar alpha) {
   return new_with_vtensor_vulkan(std::move(output), self.options());
 }
 
+Tensor& vulkan_add_(Tensor& self, const Tensor& other, Scalar alpha) {
+  auto& x = vtensor_from_vulkan(self.is_vulkan() ? self : self.vulkan());
+  auto& y = vtensor_from_vulkan(other.is_vulkan() ? other : other.vulkan());
```

Comment:
const.  I think vtensor_from_vulkan should be overloaded on const.

Reply:
I will add const override for vtensor_from_vulkan to this PR